soh/src/overlays/actors/ovl_En_Clear_Tag/z_en_clear_tag.c

```diff
@@ -475,9 +475,15 @@ void EnClearTag_Update(Actor* thisx, PlayState* play2) {
                     Math_ApproachS(&this->actor.world.rot.z, 0, 15, this->targetDirection.z);
                     Math_ApproachF(&this->targetDirection.z, 0x500, 1.0f, 0x100);
 
+                    // Introduce a range requirement in Enemy Rando so Arwings don't shoot the player from
+                    // across the map. Especially noticeable in big maps like Lake Hylia and Hyrule Field.
+                    uint8_t enemyRandoShootLaser =
+                        !CVarGetInteger("gRandomizedEnemies", 0) ||
+                        (CVarGetInteger("gRandomizedEnemies", 0) && this->actor.xzDistToPlayer < 1000.0f);
```

Comment:
You can omit the second CVarGet here to avoid two calls and because it'll always be
`!false || false && ...` or `!true || true && ...`
```suggestion
                        !CVarGetInteger("gRandomizedEnemies", 0) || this->actor.xzDistToPlayer < 1000.0f);
```

Reply:
That's what I get for trying to do it quickly. Thanks.